apps/docs/common/Navigation.js

```diff
@@ -39,7 +40,7 @@ export default function Navigation({ components }) {
       <div className={styles.Sidebar__title}>Components</div>
 
       <ul className={styles.Sidebar__list}>
-        {components.map((name) => {
+        {components.filter(component => portedComponents.includes(component)).map((name) => {
```

Comment:
this is definitely a quick hack but it'll do until I fix the component data generator from making pages without examples

Reply:
yeah, I think this is safer.  we'll eventually continue working on these.